app/src/main/java/org/luxoft/sdl_core/AndroidSettings.java

```diff
@@ -0,0 +1,89 @@
+package org.luxoft.sdl_core;
+
+import android.util.Log;
+
+public class AndroidSettings {
+
+    private static final String AndroidSection = "ANDROID";
+    private static final String TAG = CompressionUtil.class.getSimpleName();
```

Comment:
Is it designed to use CompressionUtil.class tag instead of AndroidSettings.class?

Reply:
@YaroslavLutsenko  Thanks. Its sloppy copy-paste (
 fixed in 6426040